aioxmpp/muc/service.py

```diff
@@ -618,18 +633,28 @@ def _match_tracker(self, message):
         try:
             tracker = self._tracking_by_id[message.id_]
         except KeyError:
+            key = message.from_, _extract_one_pair(message.body)
```

Comment:
Nick changes are actually better with dropping ``from_``. Stanzas are strictly ordered; the nickname we have at the time of reflection may be a different one than we think if another client concurrently changes it. However, we know which nickname we have at the time the stanza is received.